ibm/service/atracker/resource_ibm_atracker_route.go

```diff
@@ -46,7 +47,13 @@ func ResourceIBMAtrackerRoute() *schema.Resource {
 						"target_ids": {
 							Type:        schema.TypeList,
 							Required:    true,
-							Description: "The target ID List. Only 1 target id is supported.",
+							Description: "The target ID List. All the events will be send to all targets listed in the rule. You can include targets from other regions.  ",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+						"locations": {
+							Type:        schema.TypeList,
```

Comment:
After checking atrackerv2.yaml I see locations is optional ...I think directly making it as required breaks all customer...Can you check can it be optional or optioanl/computed

Reply:
Updated, now is optional